### ISSUE TITLE
Add RNW (windows) support

### ISSUE THREAD

**Issue by namrog84** [mockingbot/react-native-zip-archive#217]
A variety of RN packages have started adding windows support via React-Native-Windows

https://github.com/microsoft/react-native-windows

e.g.
https://github.com/react-native-community/react-native-video/tree/master/windows

React-Native-Zip-Archive should too. :thumbsup:

**Comment by Slyther** [comment]
is there any interest in having this merged/worked on? A few things have changed but there is still a solid need for official support @plrthink 
I'm working on making the windows version that @namrog84 has worked on (later improved upon by @nejcet here https://github.com/nejcet/react-native-zip-archive) compatible with RN 0.70.3 & RNW 0.70.3 (recently updated the repository to be up to date with mockingbot/master) but I'm probably out of my depth.

https://github.com/Slyther/react-native-zip-archive

**Comment by namrog84** [comment]
For more context of what we ended up doing for our project. 
We ended up managing to fully encapsulate all our particular and simple use cases using a RN file manager(modded RNFS) and using [jszip](https://www.npmjs.com/package/jszip).  So our need of react-native-zip-archive ended and thus we removed our usage of miniz and other libs.   We typically were just zipping a small txt file which a pure js lib was fine for us, but I'm sure those needing more native performance, they'd come looking here. 

However, I still think react-native-zip-archive would have benefit for those that need more fully featured scenarios. 
I will leave that PR open, as we did use that approach for 1-2 years without issues.  Feel free to close if no longer relevant. Otherwise I'd be glad to offer any insights or thoughts if needed.
